test/unit/Reflection/ReflectionClassConstantTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Roave\BetterReflectionTest\Reflection;
+
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\SourceLocator\Type\ComposerSourceLocator;
+use Roave\BetterReflectionTest\Fixture\ExampleClass;
+
+class ReflectionClassConstantTest extends \PHPUnit_Framework_TestCase
+{
+    private function getComposerLocator() : ComposerSourceLocator
+    {
+        global $loader;
```

Comment:
Can you try `$loader = require __DIR__ . '/../../../lots-of-../../vendor/autoload.php';`?